chia/_tests/util/blockchain.py

```diff
@@ -94,7 +94,7 @@ def persistent_blocks(
                 # if this assert fails, and changing the test chains was
                 # intentional, please also update the test chain cache.
                 # run: pytest -m build_test_chains
-                for i in range(5):
+                for i in range(40):
```

Comment:
This is the key change to validate more blocks.

Reply:
Can we give this `40` values in these two places a name and a comment that explains why we chose it? something like needing to have a certain minimum of transaction blocks as the first mismatch only shows up after several.

Reply:
addressed in the most recent commit